cfme/containers/provider/openshift.py

```diff
@@ -20,10 +20,15 @@ class OpenshiftProvider(ContainersProvider):
     db_types = ["Openshift::ContainerManager"]
 
     def __init__(self, name=None, credentials=None, key=None,
-                 zone=None, hostname=None, port=None, provider_data=None, appliance=None):
+                 zone=None, hostname=None, port=None, provider_data=None, appliance=None,
+                 sec_protocol=None):
         super(OpenshiftProvider, self).__init__(
             name=name, credentials=credentials, key=key, zone=zone, hostname=hostname, port=port,
-            provider_data=provider_data, appliance=appliance)
+            provider_data=provider_data, appliance=appliance, sec_protocol=sec_protocol)
```

Comment:
Also why are we passing sec_protocol to ContianersProvider if it's being saved within this class (as is)?